infrastructure/helm/login-setup-2.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+RELEASENAME=$1
+CLIENTSECRET=$2
+LOCALIP=$(ifconfig | grep 'inet ' | sed '/127/d' | cut -f 2 -d ' ')
+KEYCLOAKPORT=$(kubectl get svc $RELEASENAME-keycloak -o=jsonpath='{.spec.ports[0].nodePort}')
+
+# add 4180 port to $RELEASENAME-kuberay-head-svc
+kubectl patch svc $RELEASENAME-kuberay-head-svc  -p '{"spec":{"ports": [{"port": 4180,"name":"proxy","appProtocol":"tcp","Protocol":"TCP","targetport": 4180}]}}'
+
+# update $RELEASENAME-kuberay
+sed "s/<CLIENT_SECRET>/$CLIENTSECRET/g" quantumserverless-kuberay-patch.yaml | sed "s/<NODEPORT>/$KEYCLOAKPORT/g" | sed "s/<LOCALIP>/$LOCALIP/g" | sed "s/<RELEASE>/$RELEASENAME/g" > patch.yaml
+kubectl patch rayclusters.ray.io $RELEASENAME-kuberay --type=merge  --patch-file patch.yaml
+
+# restart $RELEASENAME-kuberay-head-xxx pod
+kubectl delete pod -l "ray.io/node-type=head"
```

Comment:
```suggestion
CLIENTSECRET=$2
NAMESPACE=${3:-ray}
LOCALIP=$(ifconfig | grep 'inet ' | sed '/127/d' | cut -f 2 -d ' ')
KEYCLOAKPORT=$(kubectl -n $NAMESPACE get svc $RELEASENAME-keycloak -o=jsonpath='{.spec.ports[0].nodePort}')

# add 4180 port to $RELEASENAME-kuberay-head-svc
kubectl -n $NAMESPACE patch svc $RELEASENAME-kuberay-head-svc  -p '{"spec":{"ports": [{"port": 4180,"name":"proxy","appProtocol":"tcp","protocol":"TCP","targetPort": 4180}]}}'

# update $RELEASENAME-kuberay
sed "s/<CLIENT_SECRET>/$CLIENTSECRET/g" quantumserverless-kuberay-patch.yaml | sed "s/<NODEPORT>/$KEYCLOAKPORT/g" | sed "s/<LOCALIP>/$LOCALIP/g" | sed "s/<RELEASE>/$RELEASENAME/g" > patch.yaml
kubectl -n $NAMESPACE patch rayclusters.ray.io $RELEASENAME-kuberay --type=merge  --patch-file patch.yaml

# restart $RELEASENAME-kuberay-head-xxx pod
kubectl -n $NAMESPACE delete pod -l "ray.io/node-type=head"

```

Reply:
Thanks! I put a statement in the instruction to set the default namespace to the install target namespace before start install steps.  I want to keep in that way for now.